kcptun.rb

```diff
@@ -0,0 +1,72 @@
+class Kcptun < Formula
+  desc "extremely simple & fast udp tunnel based on kcp protocol"
+  homepage "https://github.com/xtaci/kcptun"
+  url "https://github.com/xtaci/kcptun.git",
```

Comment:
Not quite. Homebrew builds from source itself & offers our pre-compiled binaries to users, whereas Cask uses upstream binaries exclusively. The archive tarball on GitHub is simply an automatically generated thing that GitHub does when tags are created.